latex.html.markdown

```diff
@@ -0,0 +1,133 @@
+---
+language: latex
+contributors:
+    - ["Chaitanya Krishna Ande", "http://icymist.github.io"]
+filename: learn-latex.tex
+---
+
+LaTeX is known to create aesthetically pleasing documents without you worrying
+about the formatting. It is also great if one wants to create documents
+containing a lot of mathematics. Getting a good document is very easy, but
+getting it to behave exactly the way you want can be a bit hairy.
+
+
+```latex
+% All comment lines start with %
+% There are no multi-line comments
+
+% LaTeX is NOT a ``What You See Is What You Get'' word processing software like
```

Comment:
You are using a combination of `` and '' here. Do 'What You See Is What You Get'